### DISCUSSION TITLE
Model List and function

### DISCUSSION THREAD

**Discussion by marshalleq** [Anjok07/ultimatevocalremovergui#705] [General]
Kind of  a basic question, but haven't found an answer - is there a list that shows what each model is intended for / been trained on?  Seems kind of crazy to guess...

Thanks.

**Comment by lulanek** [comment]
then i am crazy too and would like to know...

**Comment by Beatwrecka** [comment]
I need this too! Amazing tool and I've had great results but I'd love some more info instead of blindly trying things or trying what others say works for them.
